internal/cli/atlas/deployments/setup.go

```diff
@@ -253,14 +294,6 @@ func (opts *SetupOpts) configureMongot(ctx context.Context, keyFileContents stri
 	return err
 }
 
-func connString(port int) string {
-	return fmt.Sprintf("mongodb://localhost:%d/admin", port)
-}
-
-func (opts *SetupOpts) seed(port int, script string) error {
-	return mongosh.Exec(opts.debug, connString(port), "--eval", script)
```

Comment:
changed with mongo driver as the setup was failing when mongosh was not installed